### DISCUSSION TITLE
@siteimprove/alfa-examples is empty

### DISCUSSION THREAD

**Discussion by scttjhnsn** [Siteimprove/alfa#671] [Questions] [answered]
Any idea when the new [@siteimprove/alfa-examples](https://github.com/Siteimprove/alfa-examples) repo will be updated with examples?

I ask because I'm currently working on creating accessibility tests using [@siteimprove/alfa-cypress](https://github.com/Siteimprove/alfa-examples) and could use a little help. Simple tests that should clearly fail are actually passing.

**Comment by kasperisager** [comment, marked as answer]
Hi there and thanks for reaching out! I'm afraid I don't have an ETA on when the examples that previously resided in the core Alfa repository will resurface in the new repository. As both internal and external (third-party) APIs have shifted quite a bit over the past year, we started seeing more and more issues pop up with our current integrations and associated examples and so decided to move the examples out of the core repository.

Before we can progress on the integrations, however, we need a solution to #270 and I don't yet have an estimate of when, or even if, this work will be given priority. The progress I've made so far can be found in #287, which is currently lacking pretty far behind `master`.

**Reply by kasperisager** [reply]
An example of how to use the Cypress integration is now available here: https://github.com/Siteimprove/alfa-examples/tree/main/end-to-end-testing/cypress. I've also added a note to the README that the integrations we currently provide are still experimental and subject to change. We of course welcome any and all feedback to help shape the integrations!
